assets/base/i18n/en.json

```diff
@@ -1511,9 +1511,11 @@
   "mobile.loading_channels": "Loading Channels...",
   "mobile.loading_members": "Loading Members...",
   "mobile.request.invalid_response": "Received invalid response from the server.",
+  "mobile.login_options.chooseTitle": "Choose your login method",
```

Comment:
What case convention are we using for our i18n ids? For existing strings in the webapp, we were using snake case. For those that have been added in this repo, we've been using a mix of that and camel case. In this particular one we're using both.
cc @hmhealey @jarredwitt